debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsStreamProducer.java

```diff
@@ -371,7 +373,7 @@ private boolean schemaChanged(List<PgProto.DatumMessage> messageList, Table tabl
         // go through the list of columns from the message to figure out if any of them are new or have changed their type based
         // on what we have in the table metadata....
         return messageList.stream().filter(message -> {
-            String columnName = message.getColumnName();
+            final String columnName = PostgresTableIdTransformer.INSTANCE.fromSqlQuoted(message.getColumnName());
```

Comment:
Yes, I'm arriving at the same conclusion. The PG driver unfortunately is a bit inconsistent, we'll don't see quotes when obtaining the schema metadata, but then actual messages from the logical decoding plug-in do contain the quote characters. Hence we also cannot easily store the transformer within the id instances obtained upfront once.